addon/services/ember-freestyle.js

```diff
@@ -22,6 +21,23 @@ export default Ember.Service.extend({
 
   notFocused: computed.not('focus'),
 
+  ensureHljs() {
+    if (!hljsPromise) {
+      hljsPromise = new Promise((resolve) => {
+        return Ember.$.getScript('https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.11.0/highlight.min.js').done((script) => {
```

Comment:
We might want to consider adding a timeout and/or offline check, so that styleguides are usable offline (without syntax highlighting).